catalogue_pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/singlepagemiro/SierraMiroMergeRule.scala

```diff
@@ -17,29 +19,15 @@ import uk.ac.wellcome.platform.merger.logging.MergerLogging
   *     location for the thumbnail.
   *
   */
-object SierraMiroMergeRule extends SierraMiroMerger with SierraMiroPartitioner {
-  def mergeAndRedirectWorks(works: Seq[BaseWork]): Seq[BaseWork] = {
-    partitionWorks(works)
-      .map {
-        case Partition(sierraWork, miroWork, otherWorks) =>
-          val maybeResult = mergeAndRedirectWorkPair(
-            sierraWork = sierraWork,
-            miroWork = miroWork
-          )
-          maybeResult match {
-            case Some(result) =>
-              result ++ otherWorks
-            case _ => works
-          }
-      }
-      .getOrElse(works)
-  }
-}
+object SierraMiroMergeRule
+    extends MergerRule
+    with SierraMiroMerger
+    with SierraMiroPartitioner
 
-trait SierraMiroMerger extends Logging with MergerLogging {
-  def mergeAndRedirectWorkPair(
+trait SierraMiroMerger extends Logging with MergerLogging with WorkPairMerger {
```

Comment:
Is this trait used anywhere else? If not, could we collapse it into the object above?